.eslintrc.yaml

```diff
@@ -0,0 +1,7 @@
+root: true
+
+rules:
+  eol-last: error
+  indent: [error, 2, {SwitchCase: 1}]
+  no-trailing-spaces: error
+  no-unused-vars: [error, {vars: all, args: none, ignoreRestSiblings: true }]
```

Comment:
There is a space before the ending `}` but the space isn't balanced on the other side.